ballerina/schemaClient.bal

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/sql;
+
+# Represents an SQL metadata client.
+isolated client class SchemaClient {
+    private final Client dbClient;
+    private final string database;
+
+    # Initializes the SchemaClient object
+    #
+    # + host - The name of where the mysql server is hosted (ex: localhost)
+    # + user - The username to access the database
+    # + password - The password to access the database
+    # + database - The name of the database to be accessed
+    # + return - A `sql:Error` or nil
+    public function init(string host, string user, string password, string database) returns sql:Error? {
```

Comment:
we need to allow passing all these params https://github.com/ballerina-platform/module-ballerinax-mysql/blob/master/ballerina/client.bal#L36